CRM/Contact/Form/Task.php

```diff
@@ -78,7 +78,7 @@ public function preProcess() {
   /**
    * Common pre-processing function.
    *
-   * @param \CRM_Core_Form_Task $form
+   * @param \CRM_Contact_Form_Task $form
```

Comment:
This doesn't seem to be true, e.g.
* https://github.com/civicrm/civicrm-core/blob/3f1f9b067dfbd041ff63d105230740eac590a865/CRM/Event/Form/Task/Register.php#L68
* https://github.com/civicrm/civicrm-core/blob/3f1f9b067dfbd041ff63d105230740eac590a865/CRM/Contact/Export/Form/Select.php#L27

Reply:
ug that's pretty gross. I've added them in - I also checked & we can type action more strictly